nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1589,22 +1580,10 @@ def _write_dedup_parsed_id(self):
             ddf = dask_cudf.read_parquet(
                 self.jaccard_pairs_path,
                 columns=[self.left_id, self.right_id],
-                blocksize="1GB",
+                blocksize="512MB",
```

Comment:
Why did we have to change this? Was this causing slowdowns?

Reply:
Yeah so this is the first dedup application before encoding and dedup on strings is a expensive operation (often with a  `16x` overhead in terms of memory). 

While testing this workflow for larger scales (>40 TB) , this is the step that blew up in memory for me . 

Long term we should run `ConnectedComponents` on as few nodes as possible as this is a communication heavy stage for maximum TCO.  This step helps ensure that .